### PR TITLE
Update method so we can include event logo when publishing

### PR DESCRIPTION
**Pull request by tophevb**
Update the GraphAPI.put_event method so we can publish the event photo/logo to
the Facebook page when publishing the event.

### CODE REVIEW

File: src/facebook.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
 import random
 import mimetypes
 import httplib
+import requests
```
Comment (phildini) [inline src/facebook.py]: If you're going to include requests, please adjust the setup.py to make sure it's a required package. Here's a pretty good way to make that happen: https://github.com/eventbrite/eventbrite-sdk-python/blob/master/setup.py

File: src/facebook.py [hunk 2]
```diff
@@ -174,27 +175,22 @@ def put_event(self, id=None, page_id=None, **data):
...
+            url = '{url}{id}/'.format(
+                url=self.url,
+                id=response['id'],
+            )
+            post_args = {}
+            post_args['access_token'] = self.access_token
+            post_args['cover_url'] = data['picture']
+            # Silently ignore errors when uploading logos so it doesn't block
+            # event publishing.  Apparently v2.4 of the Facebook API will allow
+            # us to publish the event with the picture in one POST instead of
```
Comment (phildini) [inline src/facebook.py]: Did we try to do it as one? I was under the impression we broke it in two partially to  ensure the event details were always published.

Comment (tophevb) [inline src/facebook.py]: I didn't try it as one.  The email from Facebook said we needed to get the event_id back from Facebook and then publish the image url.  And it doesn't require us to read the image and to that multipart form crap.  So I figured it's good enough for now.

File: src/facebook.py [hunk 3]
```diff
@@ -174,27 +175,22 @@ def put_event(self, id=None, page_id=None, **data):
...
+        if 'picture' in data and isinstance(response, dict) and 'id' in response:
+            # Upload the event picture to the facebook event
+            url = '{url}{id}/'.format(
+                url=self.url,
+                id=response['id'],
+            )
+            post_args = {}
+            post_args['access_token'] = self.access_token
+            post_args['cover_url'] = data['picture']
+            # Silently ignore errors when uploading logos so it doesn't block
```
Comment (npopoff) [inline src/facebook.py]: If this comment is still accurate, can you log a sentry if anything goes wrong here?

**Comment by phildini** [comment]
:ship:

**Pull request merged**